bpf/lib/nodeport.h

```diff
@@ -1616,21 +1616,12 @@ static __always_inline int encap_geneve_dsr_opt4(struct __ctx_buff *ctx, int l3_
 	}
 #endif
 
-#ifdef ENABLE_HIGH_SCALE_IPCACHE
- #ifdef IS_BPF_OVERLAY
-	src_sec_identity = ctx_load_meta(ctx, CB_DSR_SRC_LABEL);
```

Comment:
I believe this was the only user of `CB_DSR_SRC_LABEL`, so that can go away as well.